roles/sensu-check/defaults/main.yml

```diff
@@ -7,87 +7,389 @@
 # When adding extra checks to it .. the second level of
 # the dict should be named for the role that uses it and
 # the third level should be the check name.
+#
+# the tree structure is :
+# sensu_checks: (type = check)
```

Comment:
I don't like the way you're documenting the structure here,  I would document it as YAML so its not confusing at all as to the actual structure.